app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,221 @@
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import coil.compose.AsyncImage
+
+private val MovieDetailsMapper: MovieDetailsMapper = MovieDetailsMapperImpl()
+
+val movieDetailsViewState = MovieDetailsMapper.toMovieDetailsViewState(MoviesMock.getMovieDetails())
+
+@Composable
+fun MovieDetailsRoute(
+
+) {
+    val detailsViewState by remember { mutableStateOf(movieDetailsViewState) }
+
+    MovieDetailsScreen(
+        detailsViewState,
+    )
+}
+
+@Composable
+fun MovieDetailsScreen(
+    movieDetailsViewState: MovieDetailsViewState,
+) {
+    LazyColumn(
+        modifier = Modifier
+            .background(color = MaterialTheme.colors.background)
+    ) {
+        item {
+            MovieImage(movieDetailsViewState)
+        }
+
+        item {
+            MovieOverview(movieDetailsViewState)
+        }
+
+        item {
+            MovieCrewman(movieDetailsViewState)
+        }
+
+        item {
+            MovieCast(movieDetailsViewState)
+        }
+    }
+}
+
+@Composable
+fun MovieCast(
+    movieDetailsViewState: MovieDetailsViewState,
+) {
+    Column {
+        Text(
+            text = "Top Billed Cast",
+            fontSize = 17.sp,
+            fontWeight = FontWeight.Bold,
+            color = MaterialTheme.colors.primary,
+            modifier = Modifier
+                .padding(horizontal = MaterialTheme.spacing.medium, vertical = MaterialTheme.spacing.small)
+        )
+        LazyRow(
+            horizontalArrangement = Arrangement.spacedBy(MaterialTheme.spacing.medium),
+            modifier = Modifier
+                .padding(horizontal = MaterialTheme.spacing.medium, vertical = MaterialTheme.spacing.small)
+        ) {
+            items(movieDetailsViewState.cast.size) { actor ->
+                ActorCard(
+                    actorCardViewState = ActorCardViewState(
+                        id = movieDetailsViewState.cast[actor].id,
+                        imageUrl = movieDetailsViewState.cast[actor].imageUrl,
+                        name = movieDetailsViewState.cast[actor].name,
+                        character = movieDetailsViewState.cast[actor].character
+                    ),
+                    modifier = Modifier
+                        .size(
+                            width = dimensionResource(id = R.dimen.actor_card_width),
+                            height = dimensionResource(
+                                id = R.dimen.actor_card_height
+                            )
+                        )
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun MovieCrewman(
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    LazyVerticalGrid(
+        columns = GridCells.Fixed(3),
+        contentPadding = PaddingValues(dimensionResource(id = R.dimen.medium_spacing)),
+        userScrollEnabled = false,
+        modifier = Modifier
+            .height(dimensionResource(id = R.dimen.crewman_height))
+    ) {
```

Comment:
This isn't a good practice. You have a vertical scrollable inside another vertical scrollable. The only reasons that your app is not crashing is because you set the height, but this is not good because it might not be enough if there are more than 6 crewman (which won't be the case here) or if the font is increased in configurations of the phone. 
Use LazyHorizontalGrid instead, or you can use Row { Column{....} } since you can assume there will always be 6 crewmen so there is no need to use `LazyLayout`.